deploy/catalog/templates/catalog.yaml

```diff
@@ -15,6 +15,24 @@ spec:
         imagePullPolicy: Always
         ports:
         - containerPort: 10000
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 3
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 2
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 1
```

Comment:
Good question. imo, this is _generally_ optimal. This will take the pod out of service the very first time `/healthz` returns a non 200 response. I won't bother to explain why I feel that's the _right_ thing to do, because the point may, in fact, be moot in this specific case...

`/healthz` is the _only_ endpoint that receives HTTP traffic to begin with, so removing the pod _from service_ isn't really a valid concern here to begin with. Come to think of it... this component doesn't even _need_ a service. (I will open a separate PR to correct that.)

This conversation will be more relevant when we add the apiserver component to this repo in the future. We can discuss optimizing readiness probes then. For now, I am removing this readiness probe from the PR and leaving just the liveness probe.

Sound ok?